src/file/file.grpc.ts

```diff
@@ -149,4 +149,16 @@ export class FileMethods {
 
     return { ancestors };
   }
+
+  /**
+   * Get a folder/file size recursively
+   * @param call
+   */
+  public static async GetRecursiveSize(call: any): Promise<{ fileSize: number }> {
+    const fileID: string = call.request.id;
+    if (!fileID) throw new IdInvalidError();
```

Comment:
Maybe `FileNotFoundError` is better. 
Invalid means that the string is wrong or has bad characters.

Reply:
My mistake.